dap-launch.el

```diff
@@ -0,0 +1,56 @@
+;; dap-mode.el --- support launch.json -*- lexical-binding: t -*-
```

Comment:
dap-launch.

Also, all variables/functions should be prefixed with dap-launch/dap-variables

Reply:
Fixed both issues in latest master.